src/Table/HeaderCell.less

```diff
@@ -0,0 +1,43 @@
+@import "../colors.less";
+
+
+.Table--header--cell {
+  background-color: @neutral_off_white;
+  border-bottom: 3px solid @neutral_silver;
+  color: @neutral_black;
+  font-size: 16px;
```

Comment:
actually, thought clever-common was setting the root to 14px, but it just leaves it as the default, so i'll leave this at 1rem and switch the body to 0.75rem and see what that looks like